_inc/client/plans-prompt/index.jsx

```diff
@@ -0,0 +1,74 @@
+/**
```

Comment:
Should this whole file sit within `_inc/client/plans`?

Reply:
I'm not sure. Would it be okay if a component `_inc/client/plans/plans-prompt.jsx` used it's 'parent' located in `_inc/client/plans/index.jsx` as a sub-component?

I thought it might be a bit counter-intuitive that's why the `plans-prompt` is located in a separate folder but I'm open to move it to `plans`.

Reply:
I don't have a strong opinion on this, it can stay where you put it 👍